cmd/poktrolld/cmd/config.go

```diff
@@ -72,6 +85,18 @@ func initAppConfig() (string, interface{}) {
 	// srvCfg.MinGasPrices = "0stake"
 	// srvCfg.BaseConfig.IAVLDisableFastNode = true // disable fastnode by default
 
+	// TODO_MAINNET: reconsider values, especially MinGasPrices and MaxTxs
+	// Set default values so `poktrolld init` creates configs with "blessed" values:
```

Comment:
```suggestion
	// Set default values so `poktrolld init` creates configs with "blessed" values:
	// BUG_DEV_NOTE: See `config.yml` for details why these have to to be set manually
```

Reply:
`config.yml` only changes values for the first validator. We use this on LocalNet only.